test/apify/crawlers/browser_crawler.test.ts

```diff
@@ -867,7 +881,7 @@ describe('BrowserCrawler', () => {
             await browserCrawler.run();
 
             delete process.env[ENV_VARS.PROXY_PASSWORD];
-            stub.restore();
+            stub.mockClear();
```

Comment:
fyi we generally need to restore, not just clear the mocks. FYI there is `jest.restoreAllMocks()` we can run in beforeEach hook and just forgot about this (could be even part of some global test setup).